docs/EN_US/ContainerizedHPCC/ContainerizedMods/RemoteAccess.xml

```diff
@@ -0,0 +1,399 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE section PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<sect2 id="RemoteAccess" role="brk">
+  <title>Create Custom Configuration Charts for Remote File Access</title>
+
+  <para>This section shows a technique to configure two separate HPCC Systems
+  Platform environments, using Helm, where one environment (env1) exposes
+  access to its logical files via the Distributed File System (DFS) service to
+  a separate environment (env2).</para>
+
+  <para>These could be in separate Kubernetes (K8s) instances or in separate
+  namespaces in the same K8s. For simplicity, this section will use two
+  separate namespaces within the same K8s.</para>
+
+  <para>This section assumes you have a copy of default setting values saved
+  to a file using:</para>
+
+  <para><programlisting>helm show values hpcc/hpcc &gt; myvalues.yaml</programlisting></para>
+
+  <sect3>
+    <title>Setting up the first environment (env1)</title>
+
+    <para><orderedlist>
+        <listitem>
+          <para>Create <emphasis role="bold">env1</emphasis> namespace (if it
+          does not already exist).</para>
+
+          <para><programlisting>kubectl create ns env1</programlisting></para>
+        </listitem>
+
+        <listitem>
+          <para>Switch to the <emphasis role="bold">env1</emphasis>
+          namespace.</para>
+
+          <para><programlisting>kubectl config set-context --current --namespace=env1</programlisting></para>
+        </listitem>
+
+        <listitem>
+          <para>Add the jetstack repo and install the jetstack certificate
+          manager.</para>
+
+          <para><programlisting>helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm install cert-manager jetstack/cert-manager --set installCRDs=true</programlisting></para>
+        </listitem>
+
+        <listitem>
+          <para>Copy the entire <emphasis role="bold">certificates:</emphasis>
```

Comment:
hm, this has diverged from the PR that this replaces....
https://github.com/hpcc-systems/HPCC-Platform/pull/16378/files , doesn't have text talking about copying sections out into separate yaml's ..

Whilst not bad to do so, it means the command lines that follow are now wrong.

@JamesDeFabia - was this a delberate change between the old PR and this replacement one?

Reply:
@jakesmith 
I thought the entire certificates section needed to be included in the custom yaml to support everything that may be in the default values yaml. 
Should I remove this or fix the command lines that follow?

Reply:
fwiw, I don't have strong preferences on whether the instructions say use the original values copy (saved with helm show ..), or whether it creates 1 new custom yaml, or whether it creates multiple new yamls, e.g. certificates.yaml and esp.yaml.
As long as it's clear which files are involved, so best to say which [example] filenames the sections are being edited or copied to, and that those filenames match up with what's being listed on the helm install line.